### PR TITLE
Cosmetic Changes to the UI

### PR DESCRIPTION
**Pull request by lalitpatel**
### Summary of changes
1. Changed the font to Merriweather
2. Added a paper background to match the game's theme
3. Changed font and color for the footer credits
4. Changed font-sizes for better readability
5. Removed outer borders to increase viewport area

### Before
image

### After
image

PS: Love the work that you guys do! 👋

### CODE REVIEW

**Comment by HSZemi** [comment]
Thank you for your proposal! There are a few things I like about it.

@HeavenlyChorus, you asked for Georgia as the default font in https://github.com/SiegeEngineers/aoe2techtree/pull/26, would Merriweather as a web font work for those cases as well?

I dislike the Arial look of the text below the icons. The game uses a serif font there as well. Would Merriweather work for those texts as well?

The paper background looks like it is not from the tech tree, but the one from the book in the campaign "cutscenes". Is that intentional?

**Comment by HeavenlyChorus** [comment]
@HSZemi as long as u specify the font in css it's fine

**Comment by lalitpatel** [comment]
THanks, @HSZemi and @HeavenlyChorus 

I am working on getting exact assets from the AOE2 DE tech tree. But it will need some image processing work. In the mean time, this background makes it look much better :)

Merriweather can for those boxes as well. I change the font in the next PR.

**Comment by lalitpatel** [comment]
I have updated the PR to change the font for unit / building names to Merriweather / serif

**Comment by lalitpatel** [comment]
There is another version of the background which is from AOE HD, and is darker and makes the text and images popup better. I like this one better. Let me know which one do you like.

dark_bg

**Comment by HSZemi** [comment]
I must say I like that one a lot better :+1:

File: index.html [hunk 1]
```diff
@@ -58,23 +59,22 @@
                 <p id="civtext"></p>
             </div>
             <div class="info">
-                <div class="info__margin">
-                    Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-36906/" target="_blank" title="Changelog">36906</a>
-                </div>
-                <div class="info__margin">
+                <p>
+                    Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-36202/" target="_blank" title="Changelog">36202</a>
```
Comment (HSZemi) [inline index.html:63]: That seems to have been merged incorrectly

**Comment by lalitpatel** [comment]
> Also, closing the help text by clicking the background does not work anymore. That is a big issue.

I figured this is due to us removing the background color for the tech-tree rect. I will fix it.

File: index.html [hunk 2]
```diff
@@ -153,7 +153,7 @@
 
         const draw = SVG('techtree').size(tree.width, tree.height)
 
-        draw.rect(tree.width, tree.height).attr({fill: 'none'}).click(hideHelp);
+        draw.rect(tree.width, tree.height).attr({fill: '#e7c28e00'}).click(hideHelp);
```
Comment (HSZemi) [inline index.html:156]: I guess that works :sweat_smile:

Comment (lalitpatel) [reply index.html:156]: Yes! Setting the fill as none makes the rect not respond to clicks. Made the background fill with opacity as 0% to hide the color while still responding to clicks. 😄

**Comment by HSZemi** [comment]
Thanks again!

**Comment by paulirish** [comment]
love these changes! Great stuff @lalitpatel

**Pull request merged**